version.sbt

```diff
@@ -1,2 +1,2 @@
 
-version in ThisBuild := "0.0.13"
+version in ThisBuild := "0.0.14-SNAPSHOT"
```

Comment:
snapshot? I don't think that is needed. We're now working towards release 0.0.14, but it won't be released until explicitly manually triggered.